### ISSUE TITLE
Sorting Articles is broken in certain cases.

### ISSUE THREAD

**Issue by wonderkindstudios** [templaza/tz_portfolio_old#11]
Both [this page](http://demo.tzportfolio.com/tz-portfolio-page?start=0) and [this page](http://demo.tzportfolio.com/tz-portfolio-page?start=30) do not automatically sort with the sort buttons, but on [this page](http://demo.tzportfolio.com/tz-portfolio-page?start=60) the buttons sort the content, and from users personal experience, the back-end custom ordering in articles makes no difference when the front end button ordering does not work.

**Comment by templaza** [comment]
Hi,
Sort Title: is sorting with alphabet for title. You can see on http://demo.tzportfolio.com/tz-portfolio-page?start=0 and http://demo.tzportfolio.com/tz-portfolio-page?start=30 title was sorted. So when you click to Title button article will not be moved.

Sort Date: Article sorting default by Date. So It will not be change when you click to Date

On http://demo.tzportfolio.com/tz-portfolio-page?start=60 you can see when click to title button title was sort normally and article was moved.

Best Regards,
Sonle

**Comment by wonderkindstudios** [comment]
What about custom sorting by ordering in the Joomla Articles panel?  This does not work.

**Comment by corconsulting** [comment]
i am having this same issue and i have filed a ticket at the templaza website but i am getting a serious runaround and no real answers. This is a very SERIOUS problem on our website.

**Comment by hennysmafter** [comment]
agreed this issue is still happening! Also opened a ticket. Today though so need to be a bit patient.
